warrior/Framework/ClassUtils/WNetwork/warrior_cli_class.py

```diff
@@ -1550,12 +1550,16 @@ def send_command(self, command, start_prompt, end_prompt,
                         if not cmd_timedout:
                             self.target_host.timeout = 1
                             pNote(tmsg1, "debug")
-                            pNote(tmsg2, "debug")
-                            pNote(tmsg3, "debug")
                             tstamp = Utils.datetime_utils.\
                                 get_current_timestamp()
                         cmd_timedout = True
                         status = "ERROR"
+                        seconds = 60
+                        pNote(tmsg3, "debug")
+                        print_info(("Will wait {0} more seconds to get end prompt " + end_prompt)
+                                   .format(seconds))
+                        Utils.datetime_utils.wait_for_timeout(seconds)
```

Comment:
Adding this line will make the iteration wait for 60 seconds so 'if int(tdelta) >= 60:' will always pass during the first iteration itself which is wrong. We will have to look for the end prompt after first failure.

Reply:
Made the required changes.